rever.xsh

```diff
@@ -35,3 +35,5 @@ $DOCKER_INSTALL_COMMAND = ('rm -rf .cache/ __pycache__/ */__pycache__ */*/__pyca
                            './setup.py install')
 $DOCKER_GIT_NAME = 'xonsh'
 $DOCKER_GIT_EMAIL = 'xonsh@googlegroups.com'
+
+$GHRELEASE_ASSETS = ['xonsh-x86_64.AppImage']
```

Comment:
I think this needs to include the default assets

Reply:
i.e.

```py
from rever.activities.ghrelease import git_archive_asset

$GHRELEASE_ASSETS = [git_archive_asset(), 'xonsh-x86_64.AppImage']
```

Reply:
@scopatz This PR allows to change the code. Please make changes as you see this. 

I also can just put the lines from your example to the branch.

Reply:
Ahh yeah... I have no idea how to check if a PR allows edits

Reply:
```$ git push --set-upstream xxh rever_appimage
ERROR: Permission to xxh/xonsh.git denied to scopatz.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

It failed to let me push

Reply:
Hmm, strange. Ok, I've added you as writer to xxh/xonsh. Please try again.

Reply:
Thanks! I have just pushed up some changes